README.md

```diff
@@ -1,465 +1,48 @@
-<!-- START doctoc generated TOC please keep comment here to allow auto update -->
-<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
-**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
-
-- [OpenShift Projected Resource "CSI DRIVER"](#openshift-projected-resource-csi-driver)
-  - [Current status with respect to the Kubernetes CSIVolumeSource API](#current-status-with-respect-to-the-kubernetes-csivolumesource-api)
-  - [Current status with respect to the Kubernetes VolumeMount API](#current-status-with-respect-to-the-kubernetes-volumemount-api)
-  - [Current status with respect to the enhancement propsal](#current-status-with-respect-to-the-enhancement-propsal)
-    - [Vetted scenarios](#vetted-scenarios)
-    - [Excluded OCP namespaces](#excluded-ocp-namespaces)
-  - [Deployment](#deployment)
-  - [Run example application and validate](#run-example-application-and-validate)
-  - [Confirm openshift-config ConfigMap data is present](#confirm-openshift-config-configmap-data-is-present)
-  - [Reference to our blog on using this component to leverage RHEL Entitlements](#reference-to-our-blog-on-using-this-component-to-leverage-rhel-entitlements)
-  - [Some more detail and examples around those vetted scenarios](#some-more-detail-and-examples-around-those-vetted-scenarios)
-    - [What happens exactly if the Share is not there when you create a Pod that references it](#what-happens-exactly-if-the-share-is-not-there-when-you-create-a-pod-that-references-it)
-    - [What happens if you have a long running Pod, and after starting it with the Share present, you remove the Share](#what-happens-if-you-have-a-long-running-pod-and-after-starting-it-with-the-share-present-you-remove-the-share)
-    - [What happens if the ClusterRole or ClusterRoleBinding are not present when your newly created Pod tries to access an existing Share](#what-happens-if-the-clusterrole-or-clusterrolebinding-are-not-present-when-your-newly-created-pod-tries-to-access-an-existing-share)
-    - [What happens if you have a long running Pod, and after starting it with the Share present and the necessary permissions present, you remove those permissions](#what-happens-if-you-have-a-long-running-pod-and-after-starting-it-with-the-share-present-and-the-necessary-permissions-present-you-remove-those-permissions)
-
-<!-- END doctoc generated TOC please keep comment here to allow auto update -->
-
 # OpenShift Projected Resource "CSI DRIVER"
```

Comment:
Remove quotation marks. If the secret store CSI driver is a CSI driver, so is this.
```suggestion
# OpenShift Projected Resource CSI Driver
```